pennylane/qchem/hamiltonian.py

```diff
@@ -181,7 +181,7 @@ def _fermionic_hamiltonian(*args):
     return _fermionic_hamiltonian
 
 
-def diff_hamiltonian(mol, cutoff=1.0e-12, core=None, active=None):
+def diff_hamiltonian(mol, mapping="jordan_wigner", cutoff=1.0e-12, core=None, active=None):
```

Comment:
Is this a slightly separate change to the title of this PR?

Does the `mapping` arg need to be second? If it's last, then this becomes a non-breaking change.